src/Parsers/ASTAlterQuery.cpp

```diff
@@ -618,4 +631,11 @@ void ASTAlterQuery::formatQueryImpl(const FormatSettings & settings, FormatState
     }
 }
 
+void ASTAlterQuery::forEachPointerToChild(std::function<void(void**)> f)
+{
+    for (const auto & child : command_list->children)
+        child->as<ASTAlterCommand &>().forEachPointerToChild(f);
+    f(reinterpret_cast<void **>(&command_list));
```

Comment:
what does it change now that now we do `forEachPointerToChild` for children as well while previously it was done only to `command_list`?

Reply:
Yes. So basically it does nothing (since command list itself cannot be replaced with query parameter).